pennylane/transforms/defer_measurements.py

```diff
@@ -34,7 +34,7 @@ def null_postprocessing(results):
 
 
 @transform
-def defer_measurements(tape: QuantumTape) -> (Sequence[QuantumTape], Callable):
+def defer_measurements(tape: QuantumTape, **kwargs) -> (Sequence[QuantumTape], Callable):
```

Comment:
Was there a reason to not be specific about the allowed keyword arguments? Allowing any keyword argument can lead to someone misspelling something or giving a not-supported keyword argument and not getting an error.

Reply:
So that users don't attempt to pass keyword arguments to the transform directly. All the keyword arguments are provided by the custom qnode transform. It is only passed in explicitly in places like `DefaultQubit.preprocess` and `QNode.construct` where the device needs to be known by the transform.

Reply:
I'd rather leave it as is for the sake of consistency. This is also how I handled qnode arguments in the qinfo transforms.